packages/esm-patient-forms-app/src/forms/form-view.component.tsx

```diff
@@ -166,7 +166,7 @@ const FormView: React.FC<FormViewProps> = ({ forms, patientUuid, patient }) => {
               pageNumber={currentPage}
               totalItems={allForms.length}
               currentItems={results.length}
-              pageUrl={`$\{openmrsSpaBase}/patient/${patientUuid}/chart/`}
+              pageUrl={`./forms-see-all.component.tsx?patientUuid=${patientUuid}`}
```

Comment:
That is a super strange URL. Please make that a nice URL... Also make the patient UUID part of the path (i.e., a segment).

Reply:
I have updated this with a new URL